src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/MemberRepository.cs

```diff
@@ -112,8 +171,33 @@ public async Task AddMemberAsync(Member member)
         {
             using (var connection = _sqlConnectionFactory.CreateConnection())
             {
-                var sqlQuery = @"INSERT INTO Members(Id, Email, IsAfk, IsBanned, LastActivity, LastNudged, Name, PhotoName, Role, SaasUserId, Status) 
-                                 VALUES (@Id, @Email, @IsAfk, @IsBanned, @LastActivity, @LastNudged, @Name, @PhotoName, @Role, @SaasUserId, @Status)";
+                var sqlQuery = $@"
+                    INSERT INTO Members
+                    (
+                        {nameof(Member.Id)}, 
+                        {nameof(Member.Email)}, 
+                        {nameof(Member.IsAfk)}, 
+                        {nameof(Member.IsBanned)}, 
+                        {nameof(Member.LastActivity)}, 
+                        {nameof(Member.LastNudged)}, 
+                        {nameof(Member.Name)}, 
+                        {nameof(Member.PhotoName)}, 
+                        {nameof(Member.Role)}, 
+                        {nameof(Member.SaasUserId)}, 
+                        {nameof(Member.Status)}
+                    ) VALUES (
+                        @{nameof(Member.Id)}, 
+                        @{nameof(Member.Email)}, 
+                        @{nameof(Member.IsAfk)}, 
+                        @{nameof(Member.IsBanned)}, 
+                        @{nameof(Member.LastActivity)}, 
+                        @{nameof(Member.LastNudged)}, 
+                        @{nameof(Member.Name)}, 
+                        @{nameof(Member.PhotoName)}, 
+                        @{nameof(Member.Role)}, 
+                        @{nameof(Member.SaasUserId)}, 
+                        @{nameof(Member.Status)}
```

Comment:
Add {nameof(Member.IsDeleted)} after merging new version.

Reply:
We won't forget :)